clients/drcachesim/common/options.cpp

```diff
@@ -654,12 +669,16 @@ droption_t<double> op_warmup_fraction(
     "is computed after the skipped references and before simulated references. "
     "This flag is incompatible with warmup_refs.");
 
-droption_t<bytesize_t>
-    op_sim_refs(DROPTION_SCOPE_FRONTEND, "sim_refs", bytesize_t(1ULL << 63),
-                "Number of memory references to simulate",
-                "Specifies the number of memory references to simulate. "
-                "The simulated references come after the skipped and warmup references, "
-                "and the references following the simulated ones are dropped.");
+// XXX i#7230: Currently the simulators count non-marker records here: we should
+// either change that to all records to match -skip_refs, or update these docs.
+droption_t<bytesize_t> op_sim_refs(
+    DROPTION_SCOPE_FRONTEND, "sim_refs", bytesize_t(1ULL << 63),
+    "Number of records to analyze",
+    "This option is honored by certain tools such as the cache and TLB simulators and "
```

Comment:
Also repeat the thing about the framework still going through all the trace records anyway so a large trace may still take a long time.

Reply:
Done.